components/Editor.js

```diff
@@ -1,35 +1,60 @@
 import styled from "styled-components";
 import {TitleField, TextField} from "./Textfields";
 import {Button} from "./Button";
+import {useState} from "react";
+import ClosingPopup from "./ClosingMessage";
 
 export default function Editor({
   saveProjects,
   handleChangePage,
   editorContent,
+  nameTakenContent,
 }) {
+  const [finalCount, setFinalCount] = useState({
+    finalWordcount: 0,
+    closingMessage: false,
+  });
+  const [wordcount, setWordcount] = useState(editorContent.wordcount);
   const handleSave = event => {
     event.preventDefault();
     const title = event.target.title.value;
     const text = event.target.text.value;
     const id = editorContent.id;
-    handleChangePage("my projects");
-    saveProjects(title, text, id);
+    const finalWordCount = wordcount - editorContent.wordcount;
+    setFinalCount({finalWordcount: finalWordCount, closingMessage: true});
+    saveProjects(title, text, id, wordcount);
   };
   return (
     <div>
+      {finalCount.closingMessage === true &&
+      nameTakenContent.taken === false ? (
+        <ClosingPopup
+          finalCount={finalCount}
+          setFinalCount={setFinalCount}
+          handleChangePage={handleChangePage}
+        />
+      ) : (
+        <></>
+      )}
```

Comment:
You could write it also like this:

```suggestion
      {finalCount.closingMessage &&
      !nameTakenContent.taken && (
        <ClosingPopup
          finalCount={finalCount}
          setFinalCount={setFinalCount}
          handleChangePage={handleChangePage}
        />
      )}
```